isis/src/base/objs/NaifDskShape/unitTest.cpp

```diff
@@ -80,6 +80,18 @@ int main(int argc, char *argv[]) {
     success = shapeModelFromPvlShape.intersectSurface(obsPos, lookDir);
     qDebug() << "Intersection successful?     " << toString(success);
 
+    qDebug() << "Try to intersect surface at obsPos (1000,0,0) and lookDir (-1,0,0)";
+    obsPos[0]  = 1000.0;  obsPos[1]  = 0.0;   obsPos[2]  = 0.0;
+    lookDir[0] = -1.0;  lookDir[1] = 0.0;  lookDir[2] = 0.0;
+    success = shapeModelFromPvlShape.intersectSurface(obsPos, lookDir);
```

Comment:
This is calling the wrong intersectSurface function, it should call the ground point + observer position version.

Reply:
Called other function